gateway-protocol-impl/src/main/resources/gateway-service-config.json

```diff
@@ -0,0 +1,50 @@
+{
+  "methodConfig": [
+    {
+      "name": [
+        {"service": "gateway_protocol.Gateway", "method": "ActivateJobs"},
+        {"service": "gateway_protocol.Gateway", "method": "BroadcastSignal"},
+        {"service": "gateway_protocol.Gateway", "method": "CancelProcessInstance"},
+        {"service": "gateway_protocol.Gateway", "method": "CompleteJob"},
+        {"service": "gateway_protocol.Gateway", "method": "DeleteResource"},
+        {"service": "gateway_protocol.Gateway", "method": "EvaluateDecision"},
+        {"service": "gateway_protocol.Gateway", "method": "FailJob"},
+        {"service": "gateway_protocol.Gateway", "method": "ResolveIncident"},
+        {"service": "gateway_protocol.Gateway", "method": "SetVariables"},
+        {"service": "gateway_protocol.Gateway", "method": "StreamActivatedJobs"},
+        {"service": "gateway_protocol.Gateway", "method": "ThrowError"},
+        {"service": "gateway_protocol.Gateway", "method": "Topology"},
+        {"service": "gateway_protocol.Gateway", "method": "UpdateJobRetries"},
+        {"service": "gateway_protocol.Gateway", "method": "UpdateJobTimeout"}
```

Comment:
❓ I think it would be easy to forget adding new RPCs here. Any ideas how to deal with that?

Reply:
There's always a choice to make (does it go in this section or in the other). So, auto-generation is not possible. A test sounds like the way to go.